### PR TITLE
Replace remoteExecFnc with events for the Medical focus features PR

### PR DESCRIPTION
**Pull request by esteldunedain**
Replace all instances of `execRemoteFnc` from ACE_Medical

### CODE REVIEW

File: addons/medical/XEH_postInit.sqf [hunk 1]
```diff
@@ -6,9 +6,14 @@ GVAR(heartBeatSounds_Fast) = ["ACE_heartbeat_fast_1", "ACE_heartbeat_fast_2", "A
...
-["medical_woundUpdateRequest", FUNC(onWoundUpdateRequest)] call ace_common_fnc_addEventHandler;
-["interactMenuClosed", {[objNull, false] call FUNC(displayPatientInformation); }] call ace_common_fnc_addEventHandler;
+// Event handlers
+["medical_propagateWound", FUNC(onPropagateWound)] call EFUNC(common,addEventHandler);
+["medical_woundUpdateRequest", FUNC(onWoundUpdateRequest)] call EFUNC(common,addEventHandler);
+["interactMenuClosed", {[objNull, false] call FUNC(displayPatientInformation); }] call EFUNC(common,addEventHandler);
+["actionCheckBloodPressureLocal", DFUNC(actionCheckBloodPressureLocal)] call EFUNC(common,addEventHandler);
+["actionCheckPulseLocal", DFUNC(actionCheckPulseLocal)] call EFUNC(common,addEventHandler);
+//@todo: DFUNC(actionDragUnit) doesn't exist
+["actionDragUnit", DFUNC(actionDragUnit)] call EFUNC(common,addEventHandler);
```
Comment (esteldunedain) [inline addons/medical/XEH_postInit.sqf]: @Glowbal, `DFUNC(actionDragUnit)` doesn't exist. Idk how this code is supposed to work

Comment (Glowbal) [inline addons/medical/XEH_postInit.sqf]: That is a deprecated function. This should be removed.

Comment (esteldunedain) [inline addons/medical/XEH_postInit.sqf]: k

**Comment by Glowbal** [comment]
I am ok with this. I think we will test it together with the rest of the medical changes.

**Pull request closed without merging**